lib/rust/ensogl/core/src/display/style/sheet.rs

```diff
@@ -135,6 +135,7 @@ impl SheetNode {
 #[allow(missing_docs)]
 pub struct Expression {
     pub args:     Vec<Path>,
+    #[allow(clippy::type_complexity)]
```

Comment:
is it really needed here? This type does not look so complex. If clippy warns here, maybe ithere is a setting for it in clippy conf now - is there one?